.github/workflows/on-pullrequest-powershell.yml

```diff
@@ -0,0 +1,45 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
GitHub is failing to detect the file rename 🙄 . I renamed the above file so that its intent would be more clear.